src/kudu/server/server_base.cc

```diff
@@ -666,6 +667,7 @@ Status ServerBase::StartMetricsLogging() {
   return Status::OK();
 }
 
+#ifdef FB_DO_NOT_REMOVE
```

Comment:
we only need to remove the minidump deleter. The excess log file deleter might still be required.